README.md

```diff
@@ -9,26 +9,48 @@ The image below shows the architecture of these serverless functions in connecti
+Service credentials used within the serverless functions in this project are loaded as environment variables using [dotenv](https://www.npmjs.com/package/dotenv).
+Before executing the `netlify dev` command, create a `.env` file in the root directory with the following values to store the needed credentials;
 
 ```
 # Access key from Stripe to access your Stripe resources
 STRIPE_KEY=STRIPE_KEY
 
-# The product ID which users are subscribed to.
-EMAIL_PRODUCT_ID=STRIPE_KEY
+# The ID of the email price product used in billing subscribers.
+EMAIL_PRODUCT_PRICE_ID=EMAIL_PRODUCT_PRICE_ID
+
+# Your Auth0 Domain
+AUTHO_DOMAIN=AUTHO_DOMAIN
+
+# Auth0 Management Application Client ID
+MGT_CLIENT_ID=MGT_CLIENT_ID
+
+# Auth0 Management Application Client Secret
+MGT_CLIENT_SECRET=MGT_CLIENT_SECRET
+
+# Email SMTP host
+SMTP_HOST=SMTP_HOST
+
+# Email SMTP Password
+SMTP_PASSWORD=SMTP_PASSWORD
+
+# Email Sender Address
+SMTP_SENDER=SMTP_SENDER
+
+# Email SMTP Username
+SMTP_USERNAME=SMTP_USERNAME
 ```
 
 ## CI / CD Pipeline
 
+All serverless function endpoints within this Cloud API all tested using [Postman Tests](https://www.postman.com/automated-testing/). The API documentation explaining the endpoints can be found in the GitHub pages for this repository [here](https://ambianic.github.io/ambianic-subscriptions.github.io/). An exported file of the postman collection in json format is available at `./tests/postman/ambianic-functions-collection.postman_collection.json`.
 
-The three currently available serverless function endpoints are all tested using [Postman Tests](https://www.postman.com/automated-testing/). The API documentation explaining the endpoints can be found in the GitHub pages for this repository [here](https://ambianic.github.io/ambianic-subscriptions.github.io/). An exported file of the postman collection in json format is available at `./tests/postman/ambianic-functions-collection.postman_collection.json`.
-
-Continuous integration for this project is managed using [GitHub Actions](https://github.com/features/actions), driven using the steps in the `.github/workflows/ci.yaml` file. The following sensitive credentials used within the CI jobs are managed using [Github Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets);
+Continuous integration for this project is managed using [GitHub Actions](https://github.com/features/actions), driven using the steps in the `.github/workflows/ci.yaml` file.
+The following sensitive credentials are used within the CI jobs are stored using [Github Secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) and are retrieved and used within each workflow build;
 
-- STRIPE_KEY
 - GITHUB_TOKEN
```

Comment:
@vickywane 

The readme is still confusing.

I see a `## Running Locally` section and a `## CI / CD Pipeline` section, but where is the live deployment section? It needs to show the runtime deployment environment variables that Netlify needs to provide to the cloud functions. For example Stripe key, Auth0 keys and Mailgun keys.

Reply:
Basically, the credentials listed in the `Running Locally` section are used when deployed to Netlify. The only difference is the variables are stored in a `.env` file locally and are stored in Netlify when deployed.

Would it be sufficient to rename that section to `Executing Serverless Functions` and also include a text indicating that the variables are also used within Netlify?